src/octoprint/plugins/classicwebcam/static/js/classicwebcam_wizard.js

```diff
@@ -0,0 +1,31 @@
+$(function () {
+    function CoreWizardWebcamViewModel(parameters) {
+        var self = this;
+
+        self.settingsViewModel = parameters[0];
+
+        self.required = false;
+
+        self.onWizardDetails = function (response) {
+            self.required =
+                response &&
+                response.corewizard &&
+                response.corewizard.details &&
+                response.corewizard.details.webcam &&
+                response.corewizard.details.webcam.required;
+        };
+
+        self.onWizardFinish = function () {
+            if (!self.required) return;
+            if (self.settingsViewModel.streamUrl()) {
+                return "reload";
+            }
```

Comment:
Accessing legacy settings, no?

Reply:
It's the `ClassicWebcamSettingsViewModel` which points to the correct field in the plugin settings